studio/components/interfaces/Settings/Addons/ComputeInstanceSidePanel.tsx

```diff
@@ -287,14 +289,47 @@ const ComputeInstanceSidePanel = () => {
                             {option.meta?.cpu_cores ?? 0}-core ARM CPU (
                             {option.meta?.cpu_dedicated ? 'Dedicated' : 'Shared'})
                           </p>
-                          <div className="flex items-center space-x-1 mt-2">
-                            <p className="text-scale-1200 text-sm">
-                              ${option.price.toLocaleString()}
-                            </p>
-                            <p className="text-scale-1000 translate-y-[1px]">
-                              {' '}
-                              / {option.price_interval === 'monthly' ? 'month' : 'hour'}
-                            </p>
+                          <div className="flex justify-between items-center mt-2">
+                            <div className="flex items-center space-x-1">
+                              <span className="text-scale-1200 text-sm">
+                                ${option.price.toLocaleString()}
+                              </span>
+                              <span className="text-scale-1000 translate-y-[1px]">
+                                {' '}
+                                / {option.price_interval === 'monthly' ? 'month' : 'hour'}
+                              </span>
+                            </div>
+                            {option.price_interval === 'hourly' && (
+                              <Tooltip.Root delayDuration={0}>
+                                <Tooltip.Trigger>
+                                  <div className="flex items-center">
+                                    <IconInfo
+                                      size={14}
+                                      strokeWidth={2}
+                                      className="hover:text-scale-1000"
+                                    />
+                                  </div>
+                                </Tooltip.Trigger>
+                                <Tooltip.Portal>
+                                  <Tooltip.Content side="bottom">
+                                    <Tooltip.Arrow className="radix-tooltip-arrow" />
+                                    <div
+                                      className={[
+                                        'rounded bg-scale-100 py-1 px-2 leading-none shadow',
+                                        'border border-scale-200',
+                                      ].join(' ')}
+                                    >
+                                      <div className="flex items-center space-x-1">
+                                        <p className="text-scale-1200 text-sm">
+                                          ${Number(option.price * 672).toFixed(0)} - $
```

Comment:
just curious - what do 672 and 744 represent?

Reply:
672 hours = 28 days
744 hours = 31 days